### PR TITLE
fix #72821 shortlinkpay.com

### LINKED ISSUE #72821: shortlinkpay.com
### Issue URL (Anti Adblock Script)
 [https://shortlinkpay.com/v2KxA?utm_source=AutoFaucet.org](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fshortlinkpay.com%2Fv2KxA%3Futm_source%3DAutoFaucet.org)
### Comment
Username: @Michal72
> 
> --
> Наведена нижче інформація генерується автоматично. Не видаляйте її.
> 
> Referrer:
> https://shortlinkpay.com/v2KxA?utm_source=AutoFaucet.org
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js
> https://userscripts.adtidy.org/release/assistant/4.3/assistant.user.js
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.2
Browser: | Chrome
AdGuard driver: | TDI
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Russian

### PR DESCRIPTION
**Pull request by wout-codes**
#72821
@zloyden

### CODE REVIEW

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -379,7 +379,7 @@ j.gs#@##bottom
...
...n,shrtwin.com,waar.site,stfly.io,cashearn.eu,seosence.com,eio.io,cutwin.com,uii.io,tapynews.com,acorpe.com,landd.xyz,shrinkerhub.com,mm1.ink,adshort.club,techbeast.xyz,techofaqs.com,zoom-link.com,short.pe,myhealthlytips.com,shorten.sh,bestlivingroomideas.com,megalink.pro,darnto.com,adshort.org,educationvilla.xyz,glory-link.com,pingit.im,skip.az,networkingz.xyz,clik.pw,travells.xyz,techsapparel.com,reportkernel.com,uiz.io,xz2.xyz,beastvila.info,latestseoupdate.com,payskip.org,fasshi.xyz,shrinkme.io,yourlearningfactory.com,urlcloud.us,2-cars.club,fasturl.in,ex-foary.com,linkfinal.com,cryptoadfly.com,dearresult.com,rohtasfitness.com,shockinghindi.com,ultraurls.com,coin.mg,shrinkurl.org,exe.io,liclink.com,glory-link1.com,shortearn.org,linkviet.net,adsafelink.com,health7am.com,shopmoon.in,gkguj.com,enrt.eu,lkop.me,trendfaq.com,linkvietvip.com,suqdz.com,trendpick.info,abonlinekaro.com,miniurl.pw,egovtjob.in,sendsurl.com,geomica.com,adshorted.net,getlink.deadtoonsindia.net,digitallearn.online,bollyreveal.com,shrtfly.net,trendlouds.com,mysiterank.org,digitalcafez.com,shotfly.in,smwebs.xyz,adbilty.me,adsafelink.net#%#//scriptlet("set-constant", "app_vars.force_disable_adblock", "undefined")
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:382]: Add `shortlinkpay.com` to the whole section `Force Adblock`

File: EnglishFilter/sections/general_extensions.txt [hunk 2]
```diff
@@ -1293,7 +1293,7 @@ y2mate.com#%#//scriptlet("set-constant", "clickAds.isShown", "true")
 y2mate.com#%#AG_setConstant('clickAds.isShown', 'true');
 ! https://github.com/AdguardTeam/AdguardFilters/issues/45388
 !#if (!adguard_app_windows && !adguard_app_mac && !adguard_app_android)
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:1295]: I think you can remove this hint.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:382]: I did right?

Comment (zloyden) [reply EnglishFilter/sections/antiadblock.txt:382]: You did it only for the scriptlet rule. There are rules for other platforms which don't support it.
It's really near `! for platforms without JS rules support` :)

**Comment by wout-codes** [comment]
Okay? :)

**Comment by zloyden** [comment]
Yes, it's okay now. Only one question. Do you need this one `#%#//scriptlet("abort-on-property-read", "open")`?

**Comment by wout-codes** [comment]
> Yes, it's okay now. Only one question. Do you need this one `#%#//scriptlet("abort-on-property-read", "open")`?

Actually, the pop-ups are not being blocked with AdGuard Windows.  Please check @zloyden

**Comment by zloyden** [comment]
I apologize for misleading you. Most likely there are several scripts here, but as @Alex-302 [said here](https://github.com/AdguardTeam/AdguardFilters/issues/72821#issuecomment-764650412) - the website has low visitors.
In this case, we don't add rules to our filters.

**Comment by wout-codes** [comment]
> I apologize for misleading you. Most likely there are several scripts here, but as @Alex-302 [said here](https://github.com/AdguardTeam/AdguardFilters/issues/72821#issuecomment-764650412) - the website has low visitors.
> In this case, we don't add rules to our filters.

I agree. Should not the AdGuard Pop-Up Blocker block pop-ups? It blocks some pop-ups in some sites but mostly it fails.

**Comment by zloyden** [comment]
Generally, it should block them.

**Pull request closed without merging**